src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1373,6 +1373,12 @@ public override Task<T> GetFieldValueAsync<T>(int ordinal, CancellationToken can
             if (cancellationToken.IsCancellationRequested)
                 return Task.FromCanceled<T>(cancellationToken);
 
+            if (typeof(T) == typeof(Stream))
+                return (Task<T>)(object)(GetStreamAsync(ordinal, cancellationToken));
+
+            if (typeof(T) == typeof(TextReader))
+                return (Task<T>)(object)(GetStreamAsync(ordinal, cancellationToken));
```

Comment:
Why not calling `GetTextReaderAsync` here?!?

Reply:
Thanks! Fixed.